plugins/orchestrator/src/components/WorkflowInstancePage.tsx

```diff
@@ -208,17 +215,14 @@ export const WorkflowInstancePage = ({
             <Grid container item justifyContent="flex-end" spacing={1}>
               {isErrorState && (
                 <Grid item>
-                  <Tooltip
-                    title="user not authorized to execute workflow"
-                    disableHoverListener={permittedToExecute.allowed}
-                  >
+                  <Tooltip title={retriggerTooltip}>
                     <Button
                       variant="contained"
                       color="primary"
-                      disabled={!permittedToExecute.allowed}
+                      disabled={isRetriggerDisabled}
```

Comment:
does it make sense to show the button if the action can't be performed? Is it possible to hide the button?
What's your opinion? Not sure if this can improve the user experience.

Reply:
It's an UX topic discussed again and again, there are 50% people who like hiding and 50% who don't :-)

I incline to the opinion that from the user's perspective is better to know that a feature is there but can not be executed right now because of reason XY.

Hiding the button can make the user confused since "I know it was somewhere here but can not find it now. Where was it? Different page? I am confused. Do I need to do something to see it again? What is that?"

Reply:
It makes sense, thanks :)